### ISSUE TITLE
Updated manuscript: focal-cn-file-preparation status update

### ISSUE THREAD

**Issue by kgaonkar6** [AlexsLemonade/OpenPBTA-manuscript#133]
#### What section of the manuscript should be updated?

_Add the section (and subsection) needing an update._
Section: Somatic Copy Number Variant Calling (WGS samples only)
Sub section: Focal Copy Number Calling

#### Proposed changes

_What changes need to be made and/or what details are missing?_
_Explain your rationale for the update._
We added a status update to include amplification and deep deletion code as:
```
    copy_number > (2 * ploidy) ~ "amplification",
    copy_number == 0 ~ "deep deletion"
```

https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1101

_Please include a level of detail that will allow another participant to make the update._

#### Relevant literature

_If there is relevant scientific literature, put links to those items here._

**Comment by jharenza** [comment]
@runjin326 will work on this

**Comment by jaclyn-taroni** [comment]
Closed via #145
